plugins/CustomDimensions/Archiver.php

```diff
@@ -245,4 +276,15 @@ protected function cleanCustomDimensionValue($value)
         return self::LABEL_CUSTOM_VALUE_NOT_DEFINED;
     }
 
+    private function getRankingQueryLimit()
+    {
+        $configGeneral = Config::getInstance()->General;
+        $configLimit = $configGeneral['archiving_ranking_query_row_limit'];
+        $limit = $configLimit == 0 ? 0 : max(
```

Comment:
I suppose this looks good. Was wondering if the 50K ranking query limit could cause issues since we technically allow 1000 root tables and 1000 entries per subtable by default. I'm assuming this will limit the number of results shown quite a bit but that's kind of also the point of this ranking query? 

For the ranking query limit would maybe do something like 

```php
$this->rankingQueryLimit = max(configLimit, 10 * $this->maximumRowsInDataTableLevelZero);
```

just in case someone has a high `datatable_archiving_maximum_rows_custom_dimensions` set and so you can still get different results per subtable but I think also not really needed.

I'm not entirely sure what the report groups by though and how the subtables are created so could also just leave as is.

Reply:
subtables are urls, so it makes sense :+1: